src-docs/src/components/guide_page/guide_page_chrome.js

```diff
@@ -142,14 +141,17 @@ export class GuidePageChrome extends Component {
             responsive={false}
             wrap>
             <EuiFlexItem grow={false}>
-              <Link to="/" className="guideLogo" aria-label="Go to home page">
+              <EuiLink
+                to="/"
+                className="guideLogo"
+                aria-label="Go to home page">
```

Comment:
Nit: you don't need to explain that a link will take you some place
```suggestion
                aria-label="EUI home">
```